packages/rancher-monitoring/generated-changes/patch/values.yaml.patch

```diff
@@ -89,16 +89,46 @@
 +##
 +k3sServer:
 +  enabled: false
-+  metricsPort: 10249
++  metricsPort: 10250
```

Comment:
Yes, but a user who enables k3sServer should not need hardenedKubelet. k3sServer exposes hardenedKubelet metrics anyways.

This is a good point though, because we should disable kubelet monitoring when k3sServer is enabled. I'll add those to the commit.

Reply:
addressed https://github.com/rancher/charts/pull/1245/commits/9eb6b41e07fd7598c4f8ce93b28ed2655c9019c8#diff-d638a3b0d7c227da5602ff9df43914a1165db68e80244779fff063ad6b8d8d5dR8